readthedocs/api/base.py

```diff
@@ -207,7 +216,7 @@ def override_urls(self):
                 name="api_get_anchor"),
         ]
 
-    def get_anchor(self, request, **kwargs):
+    def get_anchor(self, request):
```

Comment:
I'll want to double check that `kwargs` is not required by anything that we override with